docs/netdata-cloud/organize-your-infrastructure-invite-your-team.md

```diff
@@ -1,62 +1,118 @@
-# Organize Your Infrastructure and Invite your Team
+# Spaces and Rooms
 
-Netdata Cloud works with [Spaces](#netdata-cloud-spaces) and [Rooms](#netdata-cloud-rooms). They allow you to better organize your infrastructure and provide the right access to your team.
+This guide explains how to effectively organize your infrastructure monitoring using Netdata Cloud.
 
-## Netdata Cloud Spaces
+Netdata Cloud uses two primary organizational concepts:
 
-A Space is a high-level container. It's a collaboration environment where you can organize team members, access levels and the nodes you want to monitor.
+- [Spaces](#spaces): High-level containers for your entire infrastructure.
+- [Rooms](#rooms): Flexible groupings within Spaces for specific monitoring needs.
 
-### How to organize your Netdata Cloud Environment
+## Spaces
 
-You can use any number of Spaces you want, but as you organize your Cloud experience, keep in mind that you can only add any given node to a **single** Space.
+**What is a Space?**
 
-We recommend sticking to a single Space so that you can keep all your nodes and their respective metrics in one place. You can then use multiple [Rooms](#netdata-cloud-rooms) to further organize your infrastructure monitoring.
+Space serves as your primary collaboration environment in Netdata Cloud. It allows you to:
 
-### Navigate between Spaces
+- Organize team members and manage access levels.
+- Connect nodes for monitoring.
+- Create a unified monitoring environment.
 
-You can navigate through your different Spaces by using the left-most bar of the interface. From there you can also create a new Space by clicking the plus **+** icon.
+**Key Space Characteristics**
 
-![image](https://github.com/netdata/netdata/assets/70198089/74f622ac-07bf-40c7-81ba-f3907ed16c42)
```

Comment:
@Ancairon removed this screenshot because it doesn't have the plus icon :trollface:

Reply:
Well, agree :shipit: